launcher_go/v2/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/alecthomas/kong"
+	"github.com/discourse/discourse_docker/launcher_go/v2/utils"
+	"golang.org/x/sys/unix"
+	"os"
+	"os/exec"
+	"os/signal"
+)
+
+type Cli struct {
+	Version      kong.VersionFlag `help:"Show version."`
+	ConfDir      string           `default:"./containers" help:"Discourse pups config directory." predictor:"dir"`
+	TemplatesDir string           `default:"." help:"Home project directory containing a templates/ directory which in turn contains pups yaml templates." predictor:"dir"`
+	BuildDir     string           `default:"./tmp" help:"Temporary build folder for building images." predictor:"dir"`
```

Comment:
It seems like we need to add the `./tmp` directory to `.gitignore` if we intent to keep the tmp directory within the repository path.

Reply:
gitignore is updated in one of the future commits I've got lined up

Reply:
👍